packages/bezier-react/src/components/Forms/FormHelperText/FormHelperText.tsx

```diff
@@ -39,75 +36,113 @@ forwardedRef: ForwardedRef,
   const {
     visible,
     ref,
-    Wrapper,
+    labelPosition,
     ...ownProps
   } = getProps?.(rest) ?? {
     visible: true,
     ref: noop,
-    Wrapper: React.Fragment,
+    labelPosition: 'top',
     ...rest,
   }
 
   const mergedRef = useMergeRefs(ref, forwardedRef)
 
-  const shouldRendered = useMemo(() => (
-    !isEmpty(children) && visible
-  ), [
-    visible,
-    children,
-  ])
-
-  if (!shouldRendered) { return null }
+  if (isEmpty(children) || !visible) { return null }
 
   return (
-    <Wrapper>
-      <Styled.HelperText
+    <div
+      className={classNames(
+        styles.HelperTextWrapper,
+        styles[`position-${labelPosition}`],
+      )}
+    >
+      <Text
         {...ownProps}
         ref={mergedRef}
-        forwardedAs={as}
+        as="p"
+        className={styles.HelperText}
         typo={typo}
+        align="left"
       >
         { children }
-      </Styled.HelperText>
-    </Wrapper>
+      </Text>
+    </div>
   )
 })
 
-export const FormHelperText = forwardRef(({
-  testId = FORM_HELPER_TEXT_TEST_ID,
-  color = 'txt-black-dark',
-  children,
-  ...rest
-}: FormHelperTextProps,
-forwardedRef: ForwardedRef,
-) => (
-  <BaseHelperText
-    {...rest}
-    type="info"
-    ref={forwardedRef}
-    testId={testId}
-    color={color}
-  >
-    { children }
-  </BaseHelperText>
-))
-
-export const FormErrorMessage = forwardRef(({
-  testId = FORM_ERROR_MESSAGE_TEST_ID,
-  color = 'bgtxt-orange-normal',
-  children,
-  ...rest
-}: FormErrorMessageProps,
-forwardedRef: ForwardedRef,
-) => (
-  <BaseHelperText
-    {...rest}
-    aria-live="polite"
-    type="error"
-    ref={forwardedRef}
-    testId={testId}
-    color={color}
-  >
-    { children }
-  </BaseHelperText>
-))
+/**
+ * `FormHelperText` is a component to show the description of the input element.
+ * It should be used with `FormControl` component.
+ *
```

Comment:
```suggestion
/**
 * `FormHelperText` is a component to show the description of the input element.
 *
```

단독으로도 사용할 수 있게 설계되었어요!